scripts/delayed.js

```diff
@@ -81,6 +58,32 @@ async function loadHotjar() {
   })(window, document, 'https://static.hotjar.com/c/hotjar-', '.js?sv=');
 }
 
+function loadMaze() {
+  (function autoLoadMazeFunction(m, a, z, e) {
+    let t;
+    try {
+      t = m.sessionStorage.getItem('maze-us');
+    } catch (err) {
+      console.error(err);
+    }
+
+    if (!t) {
+      t = new Date().getTime();
+      try {
+        m.sessionStorage.setItem('maze-us', t);
+      } catch (err) {
+        console.error(err);
+      }
+    }
+
+    const s = a.createElement('script');
+    s.src = `${z}?apiKey=${e}`;
+    s.async = true;
+    a.getElementsByTagName('head')[0].appendChild(s);
+    m.mazeUniversalSnippetApiKey = e;
+  })(window, document, 'https://snippet.maze.co/maze-universal-loader.js', '2852429c-8735-46e0-8655-38f2f515fa53');
```

Comment:
the ID should go to constants.json, shouldn't it?
```suggestion
  })(window, document, 'https://snippet.maze.co/maze-universal-loader.js', MAZE_ID);
```

Reply:
For this I will not extract the id since it was not requested.
Eventually if needed it can be requested

Reply:
then it's ok as it is